gui/package.json

```diff
@@ -25,6 +25,8 @@
     "plotly.js-cartesian-dist": "^2.33.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-dropzone": "^14.2.3",
+    "react-markdown": "^8",
```

Comment:
Does this PR actually depend on markdown?

Reply:
This might've been a mixup in the rebase.